tutorials/kotlin/Polygons/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.3.72'
```

Comment:
Will dependabot catch dependencies to bump that have the version number pull out like this?

Reply:
Unfortunately this won't be captured by Dependabot. There's an issue here that's been marked as stale: https://github.com/dependabot/dependabot-core/issues/1618